src/libstore/build.cc

```diff
@@ -2439,7 +2471,7 @@ void DerivationGoal::runChild()
 
             /* Our ancestry. N.B: this uses literal on folders, instead of subpath. Without that,
                you open up the entire filesystem because you end up with (subpath "/") */
-            sandboxProfile += "(allow file-read-metadata\n";
+            sandboxProfile += "(allow file-read*\n";
```

Comment:
Doesn't this allow us to read tons more stuff? I was deliberately trying to keep it super restricted. Making it `file-read*` gives us the power to list the contents of /, etc.